integration-tests/src/test/java/oracle/weblogic/kubernetes/ItExternalLbTunneling.java

```diff
@@ -576,12 +467,80 @@ void testExternalRmiAccessThruNginxHttpsTunneling() {
     runExtHttpsClient(httpsTunnelingPort, 2, false);
   }
 
+  /**
+   * Verify RMI access to WLS through routes - only for OKD cluster.
+   */
+  @EnabledIfEnvironmentVariable(named = "OKD", matches = "true")
+  @Test
+  @DisplayName("Verify RMI access WLS through Route in OKD ")
+  void testExternalRmiAccessThruRouteHttpTunneling() {
+
+    assumeFalse(WEBLOGIC_SLIM, "Skipping RMI Tunnelling Test for slim image");
+    logger.info("Installing Nginx controller using helm");
+
+    // Build the standalone JMS Client to send and receive messages
+    buildClient();
+    buildClientOnPod();
+
+    // In OKD cluster, we need to set the target port of the route to be the httpTunnelingport
+    // By default, when a service is exposed as a route, the endpoint is set to the default port.
+    int httpTunnelingPort = getServicePort(
+                    domainNamespace, clusterServiceName, "CustomChannel");
+    assertTrue(httpTunnelingPort != -1,
+             "Could not get the cluster custom channel port");
+    setTargetPortForRoute(clusterServiceName, domainNamespace, httpTunnelingPort);
+    logger.info("Found the administration service nodePort {0}", httpTunnelingPort);
+    String routeHost = clusterSvcRouteHost + ":80";
+
+    // Make sure the JMS Connection LoadBalancing and message LoadBalancing
+    // works from RMI client outside of k8s cluster
+    runExtClient(routeHost, 0, 2, false);
+    logger.info("External RMI http tunneling works for Route");
+  }
+
+  /**
+   * Verify tls RMI access to WLS through routes with edge termination - only for OKD cluster.
+   */
+  @Disabled("need to add itls key and certs")
```

Comment:
typo here: itls -> tls

Reply:
why is it disabled?

Reply:
fixed the typo. Not sure why it was Disabled, I copied it from main branch